app/src/app.css

```diff
@@ -1,5 +1,5 @@
 nav {
-  background-color: rgb(59, 130, 246);
+  background-image: linear-gradient(90deg, #3b83f6 0%, #BE38B0 100%);/*rgb(59, 130, 246); */
```

Comment:
Can this be done using tailwind?

Reply:
It can and will be, but the original banner was done just through CSS so I was going to rewrite it the tailwind way when I came back for the button after MS1. Given that it works right now and I would need to learn tailwind to fix it when I need to work on the presentation I thought it would be fine.